### PR TITLE
Switch to version of react-json-view that supports React 18 (DC-733).

### PR DESCRIPTION
**Pull request by cahrens**
## Summary of changes:
Upgraded react--json-view to a version that supports React 18 and also ran `yarn dedupe` which cleaned up quite a bit of stuff.

### Why
- We are trying to move to React 18, and the original `react-json-view` library is no longer supported. This is a fork that at least has been updated for React 18 (though is not very actively maintained either).

Recommendation for this forked version came from https://github.com/mac-s-g/react-json-view/issues/403.

### Testing strategy
- [x] Manual testing, see screenshots.

### CODE REVIEW

File: src/pages/library/DataBrowserPreview.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
+import ReactJson from '@microlink/react-json-view';
 import _ from 'lodash/fp';
```
Comment (cahrens) [inline src/pages/library/DataBrowserPreview.js:2]: Before and after, tested that copy worked (both versions have copy, though it's only showing on the second screenshot below).

image
image

File: src/pages/workspaces/workspace/jobHistory/FailuresViewer.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
+import ReactJson from '@microlink/react-json-view';
 import _ from 'lodash/fp';
```
Comment (cahrens) [inline src/pages/workspaces/workspace/jobHistory/FailuresViewer.js:2]: Before and after (clipboard not enabled on these usages):

image

image

File: src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
+import ReactJson from '@microlink/react-json-view';
 import _ from 'lodash/fp';
 import { Fragment, useRef, useState } from 'react';
 import { div, h } from 'react-hyperscript-helpers';
-import ReactJson from 'react-json-view';
```
Comment (cahrens) [inline src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js:4]: Before and after:

image

image

File: src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
+import ReactJson from '@microlink/react-json-view';
 import _ from 'lodash/fp';
```
Comment (cahrens) [inline src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js:2]: I wasn't able to get into a state where I could use the CallCacheWizard, but given that the other 3 usages of this library work I think this is pretty safe.

**Review by nawatts — APPROVED** [review]

**Review by s-rubenstein — APPROVED** [review]
LGTM

File: src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js (hunk 4 above)
Comment (cjllanwarne) [reply src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js:2]: I agree. Thanks for the heads up!

**Pull request merged**